index.html

```diff
@@ -564,20 +683,23 @@ <h4>Script Type</h4>
                 the document represents a translation of the <a>Original Language Transcript</a> in a common language.</p>
                 <p>It can be adapted to produce a <a>Pre-Recording Script</a>,
                   and/or used as the basis for a further translation into the <a>Target Recording Language</a>.</p> 
-                <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain <a>Text</a> objects whose
-                  <a>Text Language Source</a> is set to <a>Translation</a>.
-                  They MAY also contain <a>Original</a> language <a>Text</a>.</p>
-                <aside class="note">A <a>Script Event</a> in this type of <a>transcript</a> has a 
-                  <a>Translation</a> <a>Text</a> object in the <a>Target Recording Language</a>
-                  and can also contain an <a>Original</a> <a>Text</a> object, for context,
-                  to assist in the adaption process.
-                  It can also have <a>Translation</a> <a>Text</a> objects in other languages,
-                  for example used as intermediate (&quot;pivot&quot;) translations,
-                  or as additional context to assist in the adaptation process.</aside>
+                <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain
+                  <a>Translation </a><a>Text</a> objects, marked as such by having their
+                  <a>Text Language Source</a> set to the source language from which they were translated,
+                  and their language set to their actual language.
+                  They MAY also contain <a>Original</a> <a>Text</a> objects.</p>
                 <aside class="example">If a programme contains dialogue in English and Hebrew,
                   the French <a>Translated Transcript</a> will contain at least the translation in French of all <a>Script Events</a>.
                   It may still retain text content in Hebrew and English to assist further processing.</aside>
-              </li>
+                <aside class="example">If an <a>audio description</a> <a>Original Language Transcript</a> contains
+                  <a>Original</a> language <a>Script Events</a> that describe in-image text content,
+                  and the desired <a>audio description</a> output needs to be in
+                  a different language, the <a>Text</a> objects can be translated and marked as <a>Translation</a>.
+                  In that case, it is appropriate to mark the transcript as a <a>Translated Transcript</a>.
+                  It may also be appropriate, as part of the translation activity, to describe within the translated text
+                  the original language of that text, to inform the audience.
```

Comment:
It may be good to illustrate this case with an example. For me the meaning of `describe within the translated text the original language of that text` war not clear.

Reply:
Done in 10b1a0e